docs/Storm-Scheduler.md

```diff
@@ -4,7 +4,7 @@ layout: documentation
 documentation: true
 ---
 
-Storm now has 4 kinds of built-in schedulers: [DefaultScheduler]({{page.git-blob-base}}/storm-server/src/jvm/org/apache/storm/scheduler/DefaultScheduler.java), [IsolationScheduler]({{page.git-blob-base}}/storm-server/src/jvm/org/apache/storm/scheduler/IsolationScheduler.java), [MultitenantScheduler]({{page.git-blob-base}}/storm-server/src/jvm/org/apache/storm/scheduler/multitenant/MultitenantScheduler.java), [ResourceAwareScheduler](Resource_Aware_Scheduler_overview.html). 
+Storm now has 4 kinds of built-in schedulers: [DefaultScheduler](../storm-server/src/main/java/org/apache/storm/scheduler/DefaultScheduler.java), [IsolationScheduler](../storm-server/src/main/java/org/apache/storm/scheduler/IsolationScheduler.java), [MultitenantScheduler](../storm-server/src/main/java/org/apache/storm/scheduler/multitenant/MultitenantScheduler.java), [ResourceAwareScheduler](./Resource_Aware_Scheduler_overview.md). 
```

Comment:
Let's leave `{{page.git-blob-base}}` as it will be replaced with github url.

Please refer links in http://storm.apache.org/releases/1.0.3/Storm-Scheduler.html